modules/claims_api/app/models/claims_api/evss_claim.rb

```diff
@@ -56,5 +56,44 @@ def supporting_documents
         []
       end
     end
+
+    def self.services_are_healthy?
+      # TODO: we should add check for Okta and SAML Proxies being up as well
+      last_mvi_outage = Breakers::Outage.find_latest(service: MVI::Configuration.instance.breakers_service)
+      mvi_up = (last_mvi_outage.blank? || last_mvi_outage.end_time.present?)
+
+      last_evss_claims_outage = Breakers::Outage.find_latest(service: EVSS::ClaimsService.breakers_service)
+      evss_claims_up = last_evss_claims_outage.blank? || last_evss_claims_outage.end_time.present?
+
+      last_evss_common_outage = Breakers::Outage.find_latest(service: EVSS::CommonService.breakers_service)
+      evss_common_up = last_evss_common_outage.blank? || last_evss_common_outage.end_time.present?
+      mvi_up && evss_claims_up && evss_common_up
+    end
+
+    def self.healthy_service_response
+      {
+        data:  {
+          id: 'claims_healthcheck',
+          type: 'claims_healthcheck',
+          attributes: {
+            healthy: true,
+            date: Time.zone.now.to_formatted_s(:iso8601)
+          }
```

Comment:
should these attributes (every key in `data`, actually) also be in the `unhealthy_service_response`? they still seem applicable

Reply:
Hey @mjlumetta they do not seem to be present on the error response of the appeals healthcheck error response. Which was our starting point. @bastosmichael and @dvogel what do you think? I am open to adding them.

Reply:
yeah, that makes sense if that's what you're working from, and i know pretty much nothing about the specific context, so i defer to whatever you think is best. i guess my position on this is this (and it's like, a tiny comment): if you're returning both of these JSON structures from the same endpoint, then i don't get the value of `data.attributes.healthy`; when would you ever check it? the healthy vs unhealthy check for a consumer would be satisfied by the 200 vs 503 response and `res.data` vs `res.errors` as the top level key, so a `healthy` boolean nested that deep doesn't add anything. similarly, why have an `id`/`type` field in only one fork of the response? either i need that info to process at some point, in which case i might need it for healthy or unhealthy use cases, or i don't (bc i know that i'm calling the healthcheck endpoint, so i can just decide how to identify that as a consumer). 

it's very possible that there are specific reasons for these choices and that's totally cool if there are. but it's difficult to understand what this means in context intuitively because there's a lot of divergence between responses.

Reply:
@mjlumetta  the data structure was put in place to mimic the appeals healthcheck which we don't control. Which a passthrough to a VA endpoint. The goal was consistency, but there is nothing really tying this code to it other than having the healthchecks be consistent The ID/Type are in place for json api compliance.

Reply:
cool, that makes sense! i mean, as long as there's a reason. thanks for looking into it, Charley